src/server/cluster/outgoing_slot_migration.cc

```diff
@@ -17,8 +17,7 @@
 #include "server/journal/streamer.h"
 #include "server/server_family.h"
 
-ABSL_FLAG(int, source_connect_timeout_ms, 20000,
-          "Timeout for establishing connection to a source node");
+ABSL_FLAG(int, connection_timeout_ms, 2000, "Timeout for network operations");
```

Comment:
interface needs int type, maybe in the future will change that

Reply:
It's fine to keep as is, I just want to understand why. Which interface required int? Flags? Because absl flags support duration.
We can't change that in the future, once it's released to the public :)

Reply:
not flags, timeout call needs int

Reply:
`ToInt64Milliseconds()`?

Reply:
OK)